### PR TITLE
Addition of issue #14 - Checking the follwer count

### PR DESCRIPTION
**Pull request by littleswany**
Adding functionality that checks the numbers of followers a user has
and compares this to a user set value and if this exceeds the value no
further interaction happens.

Command:
.set_upper_follower_count(limit = 250) \

Addition of this to documentation and readme also

### CODE REVIEW

File: instapy/instapy.py [hunk 1]
```diff
@@ -184,6 +186,13 @@ def clarifai_check_img_for(self, tags=None, comment=False, comments=None):
 
     return self
 
+  def set_upper_follower_count(self, limit=100):
+    """Used to chose if a post is liked by the number of likes"""
+    if limit is None:
+      self.like_by_followers_limit = 10000000000
+    self.like_by_followers_limit = limit
+    return self
```
Comment (AlainPilon) [inline instapy/instapy.py]: 190 to 193 could be replaced with:
`self.like_by_followers_limit = limit or 1000000000`

**Comment by pekapa** [comment]
If I might make a suggestion...

Instead of using a huge number (10000000000), use 0 and add and `if like_by_followers_limit`
Also, the way your are parsing the `.set_upper_follower_count` the user can only set it to `None` if he explicitly do so (change de default `limit` value to `None`). Finally, if it is `None`, the code does break as you set it to `None` and not the huge number (int > None always evaluate to True).

**Comment by littleswany** [comment]
Thanks @pekapa.

@pekapa so are you suggesting to have the default value as 0 and then checking for a 0 value or setting the default to None?

File: instapy/instapy.py (hunk 1 above)
Comment (littleswany) [reply instapy/instapy.py]: Thanks @AlainPilon will look into this

**Comment by pekapa** [comment]
I'm suggesting to have the default as 0 and in than:

```python
if like_by_followers_limit and (num_followers > like_by_followers_limit):
    return True, user_name
```

no need to use `None`

**Comment by littleswany** [comment]
Thanks, yeah Im with you, gimme a sec lemme just update and test

**Comment by littleswany** [comment]
@pekapa Have updated

**Comment by AlainPilon** [comment]
BTW a little bit off topic but, what was the thought process to have the functions return `True, user_name` when they should not be liked? I know the variable is `inappropriate` but shouldnt we reverse the boolean logic to make it more intuitive?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+> Feel free to contribute to the project in whatever way! <br /> If you're not familiar with python, you could build a github page for this project (Just head over to the issues, there might be a task for you). You're a marketer? Perfect! hit me with a message on contact.timgrossmann@gmail.com. <br/> If you don't have the time or skills to contribute, you can also support us through Patreon! <br /> **Become a part of InstaPy!**
 
 > **Disclaimer**: Please Note that this is a research project. I am by no means responsible for any usage of this tool. Use on your own behalf. I'm also not responsible if your accounts get banned due to extensive use of this tool.
 
-<img src="http://i.imgur.com/9ZjtveL.png" width="150" align="right">
+<img src="https://s3-eu-central-1.amazonaws.com/centaur-wp/designweek/prod/content/uploads/2016/05/11170038/Instagram_Logo-1002x1003.jpg" width="200" align="right">
```
Comment (timgrossmann) [inline README.md:6]: Please pull the master branch again and rebase on it...
We don't want to alter the readme more than your insertions 😉

File: instapy/like_util.py [hunk 3]
```diff
@@ -56,15 +57,29 @@ def check_link(browser, link, dont_like, ignore_if_contains, username):
...
+  sleep(1)
+  num_followers = browser.execute_script("return window._sharedData.entry_data.ProfilePage[0].user.followed_by.count")
+  browser.get(link)
+  sleep(1)
+
+
   print('Image from: ' + user_name)
   print('Link: ' + link)
   print('Description: ' + image_text)
+  print "Number of Followers: ", num_followers
```
Comment (timgrossmann) [inline instapy/like_util.py:78]: We only want to print this if we have set the `like_by_followers` to true, don't we?

**Review by timgrossmann — CHANGES_REQUESTED** [review]

**Comment by timgrossmann** [comment]
@littleswany This is really nice! Thank you for the collaboration!
Could you please also add lower_follower_count?
I think that shouldn't be too much work now, since you already added the upper one 😉 

> Make sure to rebase :P

File: README.md (hunk 2 above)
Comment (littleswany) [reply README.md:6]: Done, will update shortly.

File: instapy/like_util.py (hunk 3 above)
Comment (littleswany) [reply instapy/like_util.py:78]: I have removed as it is not really necessary just had it in there for my testing.

**Comment by littleswany** [comment]
@timgrossmann Glad you like it and your welcome, glad to be able to get involved!
Have committed the requested updates.

File: README.md (hunk 2 above)
Comment (timgrossmann) [reply README.md:6]: ``


sorry, I probably communicated it wrong:
This is the new image link

Also, could you please add  your additions to the index.md in /docs/?
Would be awesome!

**Review by timgrossmann — APPROVED** [review]

**Comment by littleswany** [comment]
@timgrossmann No worries, have updated!

**Comment by timgrossmann** [comment]
solved #14

**Pull request merged**